ironfish/src/mining/manager.ts

```diff
@@ -66,9 +66,21 @@ export class MiningManager {
     this.chain.onConnectBlock.on(
       (block) =>
         void this.onConnectedBlock(block).catch((error) => {
-          this.node.logger.info(
-            `Error creating block template: ${ErrorUtils.renderError(error)}`,
-          )
+          const headChanged =
+            error instanceof BlockCreationError &&
+            error.reason === BlockCreationErrorReason.HEAD_CHANGED
+
+          if (headChanged) {
+            this.node.logger.info(
+              `Chain head changed while creating block template for seqeunce ${
```

Comment:
`s/seqeunce/sequence/`

Reply:
What do you think about bumping this down to `logger.debug`? Is it useful for general users to see it in their logs?

Reply:
Yea I changed to debug